tools/st2_deploy.sh

```diff
@@ -266,11 +266,11 @@ fi
 touch $config
 cat <<mistral_config >$config
 [database]
-connection=mysql://mistral:StackStorm@localhost/mistral
-max_pool_size=50
+connection = mysql://mistral:StackStorm@localhost/mistral
+max_pool_size = 100
```

Comment:
Yeah, what is the # here? It is configurable in Puppet, but the default is `100`

This is a prime example of mismatch, and why this is needed btw.. It used to be `100`, and that's what was coded into the Puppet Module as the default. But, now it's `50`.